editor/plugins/game_view_plugin.h

```diff
@@ -93,6 +93,7 @@ class GameView : public VBoxContainer {
 		CAMERA_MODE_EDITORS,
 		EMBED_RUN_GAME_EMBEDDED,
 		EMBED_MAKE_FLOATING_ON_PLAY,
+		EMBED_DISTRACTION_FREE_MODE
```

Comment:
to mantain previous code style should have a coma `,` at the end `EMBED_DISTRACTION_FREE_MODE,`